libs/designer-ui/src/lib/unitTesting/outputMocks/ouputsSettings.tsx

```diff
@@ -39,6 +27,7 @@ export const OutputsSettings: React.FC<OutputsSettingsProps> = ({ nodeId, output
       ))}
     </>
   ) : (
-    <Text>{intlText.NO_OUTPUTS}</Text>
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    <></> // Render nothing when action result is not SUCCESS
```

Comment:
NIT: You can return null in this type of scenarios. Therefore you don't need to overwrite the eslint rule

Reply:
Changes made:

Changed the return type of the component to JSX.Element | null.
Added an early return with null when there are no mock outputs to render.
Removed the empty fragment and the ESLint disable comment.